src/tic_tac_toe/board.clj

```diff
@@ -1,4 +1,13 @@
 (ns tic-tac-toe.board)
 
-(defn empty-board []
-  '(- - - - - - - -))
+(defn- valid-location? [location board] 
+  (if (<= location (- (count board) 1))
+     true
+     false))
+
+(defn make-move [board location mark]
+  (if (valid-location? location board)
+    (assoc board location mark) board))
```

Comment:
What happens if the user inputs a character? Does the code bomb-out (I imagine undesirable!) or is it handled?